test/annotation_utils.t

```diff
@@ -1546,37 +1530,32 @@ subtest '_write_gff_from_genome' => sub {
     ok((-e $gff_fpath), "GFF file created for $obj_Echinacea.\n");
     ok ((-s $gff_fpath), "GFF file written for $obj_Echinacea.\n");
 
-    print "ALL lines of the GFF file:\n";
-    $annoutil->_print_fasta_gff(0, 2000, $gff_fpath);
+    print "First 20 lines of the GFF file:\n";
+    $annoutil->_print_fasta_gff(0, 20, $gff_fpath);
 };
 
 
 subtest 'anno_utils_rast_genome' => sub {
     # testing anno_utils rast_genome using obj ids from prod ONLY
-    my @regexes = ( qr/ERROR calling rast run_pipeline/,
-                    qr/Can\'t call method/);
-    my $allre= "(".join("|",@regexes).")";
     my ($parms, $rast_ret, $genome_obj);
     $parms = {
         "object_ref" => $obj_Ecoli,
         "output_genome_name" => "rasted_ecoli_prod",
         "output_workspace" => $ws_name
     };
-
-    throws_ok {
+    lives_ok {
         $rast_ret = $annoutil->rast_genome($parms);
-    } qr/$allre/,
-        '$annoutil->rast_genome returns ERROR due to kmer data absence or other causes.';
+    } "annoutil->rast_genome call returns normally on genome $obj_Ecoli.";
+
```

Comment:
Can you explain why these tests have gone from throwing errors to passing, without any change in the application logic?

Reply:
It was a missed update I should have made after I changed in the application logic earlier.  I made the change in one of the tests within the same subtest function, yet forgot to change the other two.